ddl/util/syncer_test.go

```diff
@@ -225,7 +225,7 @@ func TestSyncerSimple(t *testing.T) {
 }
 
 func isTimeoutError(err error) bool {
-	if terror.ErrorEqual(err, goctx.DeadlineExceeded) || grpc.Code(errors.Cause(err)) == codes.DeadlineExceeded ||
+	if terror.ErrorEqual(err, goctx.DeadlineExceeded) || status.Code(errors.Cause(err)) == codes.DeadlineExceeded ||
```

Comment:
>Code returns the error code for err if it was produced by the rpc system. Otherwise, it returns codes.Unknown.
>Deprecated: use status.Code instead.


https://godoc.org/google.golang.org/grpc#Code